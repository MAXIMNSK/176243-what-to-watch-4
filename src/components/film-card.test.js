import React from "react";
import renderer from "react-test-renderer";

import FilmCard from "./film-card.jsx";

const mock = {
  name: `Форрест Гамп`,
  picture: `https://m.media-amazon.com/images/M/MV5BNWIwODRlZTUtY2U3ZS00Yzg1LWJhNzYtMmZiYmEyNmU1NjMzXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_.jpg`,
  id: `f1`
};

describe(`Film card component total check`, () => {
  it(`Check should render film card component`, () => {
    const currentComponent = renderer.create(
        <FilmCard
          filmName={mock.name}
          picture={mock.picture}
        />
    ).toJSON();

    expect(currentComponent).toMatchSnapshot();
  });
});
