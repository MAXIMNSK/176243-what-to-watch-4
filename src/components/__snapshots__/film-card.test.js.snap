// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Film card component total check Check should render film card component 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <div
    className="small-movie-card__image"
  >
    <video
      controls={true}
      height="175"
      muted={true}
      poster="https://m.media-amazon.com/images/M/MV5BNWIwODRlZTUtY2U3ZS00Yzg1LWJhNzYtMmZiYmEyNmU1NjMzXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Форрест Гамп
    </a>
  </h3>
</article>
`;
